test/spmd/test_xla_sharding_base.py

```diff
@@ -31,6 +32,13 @@ def forward(self, x):
   def setUpClass(cls):
     cls.n_devices = xr.global_runtime_device_count()
     cls.device_ids = np.array(range(cls.n_devices))
+    xr.use_spmd()
+
+  @classmethod
+  def tearDownClass(cls):
+    del os.environ['XLA_USE_SPMD']
+    if 'XLA_AUTO_SPMD' in os.environ:
+      del os.environ['XLA_AUTO_SPMD']
```

Comment:
add a TODO here to switch to api instead of env var eventually?

Reply:
Ack, will add that under `runtine.use_spmd`. By the way, I think we should continue support both if possible -- envvar is the only way to set something system-wide, which is probably why XLA rely on this mechanism.